Components/Queues/QueueProducer.php

```diff
@@ -46,7 +47,7 @@ public function send(Exchange $ex, EndpointInterface $endpoint)
             throw new \RuntimeException("Found queue driver with name '$queueDriverName' that doesn't implement QueueDriverInterface");
         }
 
-        $queueMessage = $queueDriver->createQueueMessage();
+        $queueMessage = $queueDriver->createQueueMessage($queueName, $options);
```

Comment:
The Amqp native driver was creating the queueMessage without needed of ```$queueName``` and ```$options```. Why for the new driver is needed to pass these arguments?

Reply:
It was changed to the former approach.